plugins/eslint-config/index.js

```diff
@@ -53,6 +53,9 @@ module.exports = {
     'no-warning-comments': 'warn',
     'new-cap': ['error', {'capIsNewExceptionPattern': '^.*Error'}],
     'no-invalid-this': 'off',
+    // valid-jsdoc does not work properly for interface methods.
+    // https://github.com/eslint/eslint/issues/9978
+    'valid-jsdoc': 'off',
```

Comment:
Do we want to turn this off globally?

Reply:
This was intentional. This rule is deprecated according to: https://eslint.org/blog/2018/11/jsdoc-end-of-life (which is also linked in the github issue).
The `eslint-plugin-jsdoc` is separate and will still be checking our jsdoc (so we don't lose coverage there).